static/css/tips.css

```diff
@@ -0,0 +1,3 @@
+select {
+    display: block !important;
+}
```

Comment:
missing line

Reply:
In css files the formating is without a new line I think

Reply:
run : pre-commit run -a 
to fix all the missing lines and formatting

Reply:
Thanks